src/Chirp.Web/Pages/TimelineModel.cshtml.cs

```diff
@@ -17,97 +18,107 @@ public TimelineModel(IAuthorRepository authorRepository, ICheepService service,
         _followRepository = followRepository;
     }
 
-    public async Task<IActionResult> OnPostAsync()
+    public int GetUserId(string? authorName = null)
```

Comment:
Hvorfor defaulter vi til null her?

Reply:
for i linjen nedenunder tjekker den if authorName == null then authorName = User.Identity.Name, men måske på en ikke så læsbar måde

Reply:
Altså hvis `authorName` alligevel bliver til `User.Identity.Name` hvad afholder os så fra at gøre følgende?
```cs
public int GetUserId(string authorName = User.Identity.Name)
```